snap/snapcraft.yaml

```diff
@@ -45,14 +45,13 @@ description: |
   - Whitepaper: https://ionomy.com/en/whitepaper
   - Technical whitepaper: https://tinyurl.com/y3st8jfb
   
-  Get source code from Github: https://bitbucket.org/ioncoin/ion.git
-  Get CE source code from Github: https://github.com/cevap/ion.git
+  Get source code from Github: https://github.com/ioncoinicore/ion.git
   
   Chat with Support on Discord: https://discord.gg/vuZn7gC
   
   Contact us on social networks and forums:
   
-  - Instagram: https://t.me/ionomy
+  - Telegram:  https://t.me/ionomy
   - Twitter:   https://twitter.com/ionomics
   - Reddit:    https://www.reddit.com/r/ionomy/
```

Comment:
@korvasc should first confirm contact us on social networks and forums has valid links, We do not have access to all those links therefore those should be removed or updated

Reply:
have removed all but ioncoincore links

Reply:
have removed applicable links